spyder/app/tests/test_mainwindow.py

```diff
@@ -266,6 +266,60 @@ def test_filter_numpy_warning(main_window, qtbot):
     CONF.set('variable_explorer', 'minmax', False)
 
 
+@pytest.mark.slow
+@flaky(max_runs=3)
+@pytest.mark.use_introspection
```

Comment:
Please remove this marker. It's not needed because in Spyder 3 we only extract object info from the Console, not from the Editor (that could change in Spyder 4).